deployments/prod/environment.py

```diff
@@ -1181,6 +1181,7 @@ def mkdict(previous_catalog: dict[str, str],
     mksrc('bigquery', 'datarepo-c9158593', 'lungmap_prod_20037472ea1d4ddb9cd356a11a6f0f76__20220307_20241002_lm8'),
     mksrc('bigquery', 'datarepo-35a6d7ca', 'lungmap_prod_3a02d15f9c6a4ef7852b4ddec733b70b__20241001_20241002_lm8'),
     mksrc('bigquery', 'datarepo-131a1234', 'lungmap_prod_4ae8c5c91520437198276935661f6c84__20231004_20241002_lm8'),
+    mksrc('bigquery', 'datarepo-936db385', 'lungmap_prod_6135382f487d4adb9cf84d6634125b68__20230207_20241106_lm8'),
```

Comment:
I think this requires bumping the count.

Reply:
Negative, this is actually an update to https://github.com/DataBiosphere/azul/pull/6701/files/56c53c1a7fa0865c798600f5dcd8985bb885b0a2#diff-8f9d00d944593bab5e94f965b9d6dc374cbc696434623ed155ddf59ebf8edad2R1157.